lib/pipeline/statelessPipelineStack.ts

```diff
@@ -20,6 +20,41 @@ export class StatelessPipelineStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props: cdk.StackProps) {
     super(scope, id, props);
 
+    // Define CodeBuild project for GH action runner to use
+    // the GH repo defined below already configured to allow CB webhook
+    // This is actually not part of the pipeline, so I guess we could move this someday.
+    const ghRunnerRole = new iam.Role(this, 'GHRunnerRole', {
+      assumedBy: new iam.ServicePrincipal('codebuild.amazonaws.com'),
+    });
+    new codebuild.CfnProject(this, 'GHRunnerCodeBuildProject', {
+      // the name here act as a unique id for GH action to know which CodeBuild to use
+      // So if you change this, you need to update the GH action yaml file (.github/workflows/prbuild.yml)
```

Comment:
Huh? 
The name here does not match the name in the prbuild.yml file though. How are they matched up?

Reply:
Ahh so the pattern on the GH action side is below, so the project name should be the exact name. I'll update this on the commenting

```
runs-on: codebuild-<project-name>-${{ github.run_id }}-${{ github.run_attempt }}
```